doc/authentication-methods/client-certificate.md

```diff
@@ -0,0 +1,108 @@
+## Overview
```

Comment:
important point is missing. 
`ejabberd_auth:store_type(Host)` must be scram.  that's a bit messy and so hard to describe in terms of configuration. but generally it depends on the list of selected auth backends (some of them simply hardcode `M:store_type/1` function) and `password_format`  in  `auth_opts` section of **ejabberd.cfg** (must be set to `scram`)

Reply:
I'll provide a list of compatible backends then. :)